tutorials/cp-cf-processvisibility-config-samplescenario/cp-cf-processvisibility-config-samplescenario.md

```diff
@@ -88,49 +87,50 @@ In this step, you will import the `DesignToOperateEvents.json` to the Event Acqu
 
 In this step, you will import the `DesigntoOperate.zip` file to the Configure Business Scenarios application.
 
-1. Log on to the **Configure Business Scenarios** tile on SAP Fiori launchpad.
+1. Log on to the **Configure Visibility Scenarios** tile by navigating to the home screen.
 
-    ![config scenario](config_flp.png)
+    !![config scenario](config_flp.png)
 
-2. Choose the **Import Business Scenario** icon.
+2. Choose the **Import Scenario** icon.
 
-    ![import scenario](screen8-importbusinessscaenrio.png)
+    !![import scenario](screen8-importbusinessscaenrio.png)
 
-3. In the **Import Business Scenario** dialog, browse for the `DesigntoOperate.zip` file from your local system. Then choose **Import**.
+3. In the **Import Scenario** dialog, browse for the `DesigntoOperate.zip` file from your local system. Then choose **Import**.
 
-    ![browse](screen8-save.png)
+    !![Browse screen](screen8-browse.png)
 
     You can now see the imported business scenario as shown.
 
-      ![imported scenario](screen9-d2o.png)
+      !![imported scenario](screen9-d2o.png)
 
 4. Choose the **Design to Operate** business scenario.
 
-    ![imported scenario](screen9-d2o.png)
+    !![imported scenario](screen9-d2o.png)
 
+5. Click on **Activate**.
 
-    ![Activate](screen9-activate.png)
+    !![Activate](screen9-activate.png)
 
-    > You will see a message upon successful activation of the business scenario.
+    > You will see a message upon successful activation of the scenario.
 
 [DONE]
 [ACCORDION-END]
 
 [ACCORDION-BEGIN [Step 4: ](Process the data)]
 
-In this step, you can process the data of the activated business scenario using the Monitor Scenarios application.
+In this step, you can process the data of the activated business scenario using the Monitor Visibility Scenarios application.
 
-1. Log on to the **Monitor Scenarios** tile on SAP Fiori launchpad.
+1. Click on the **Monitor Visibility Scenarios** tile by navigating to the home screen.
```

Comment:
We don't write "click on the [thing to be clicked]". Just write "click the [thing to be clicked]".